po/fr.po

```diff
@@ -108,7 +156,9 @@ msgstr "Mode automatique ou forcer le Thème clair/foncé."
 #. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
 #: src/exm-application.c:112
 msgid "translator-credits"
-msgstr "PaxtonGround"
+msgstr ""
+"PaxtonGround"
+"Geoffrey Coulaud geoffrey.coulaud@gmail.com"
```

Comment:
This should have the email inside angle brackets (otherwise it won't display as a link)

i.e. `Geoffrey Coulaud <geoffrey.coulaud@gmail.com>`